wagtail/snippets/views/snippets.py

```diff
@@ -479,6 +507,15 @@ def get_columns(self):
             DateColumn("timestamp", label=_("Date")),
         ]
 
+    def get_breadcrumb_items(self):
+        return self.breadcrumb_items + [
+            {
+                "url": self.get_edit_url(self.object),
+                "label": get_latest_str(self.object),
+            },
+            {"label": _("History")},
```

Comment:
To turn this into a link, the view class needs to define `history_url_name` (its own URL name, basically), and we need to pass the value from the viewset.

I think with our plans of bringing these features to the generic `ModelViewSet`, this should be defined in the generic views. However, to avoid making the scope too big, I'm leaving it for later. This makes the PR focused on creating the generic breadcrumbs tag + using it to refactor the snippets breadcrumbs as an example. Further work will be done in other PRs.

Same case with the usage view.